test/usecase/simplestUsecase.js

```diff
@@ -42,17 +42,18 @@ describe('A use case', () => {
                 //when
                 await uc.run()
                 //then
+
```

Comment:
What about the `elapsedTime` for the whole use case?

Reply:
I added elapsed to usecase now.